### PR TITLE
4314: Ensured handlers are grouped by folders

### PR DESCRIPTION
**Pull request by assoomro**

### CODE REVIEW

File: src/NHSD.BuyingCatalogue.Solutions.API/Controllers/ContactDetailsController.cs [hunk 1]
```diff
@@ -6,6 +6,7 @@
 using Microsoft.AspNetCore.Mvc;
 using NHSD.BuyingCatalogue.Solutions.API.ViewModels;
 using NHSD.BuyingCatalogue.Solutions.Application.Commands.UpdateSolutionContactDetails;
+using NHSD.BuyingCatalogue.Solutions.Application.Queries.GetContactDetailBySolutionId;
 using NHSD.BuyingCatalogue.Solutions.Application.Queries.GetSolutionById;
```
Comment (sam-roberts-bjss) [inline src/NHSD.BuyingCatalogue.Solutions.API/Controllers/ContactDetailsController.cs]: Is this using still referenced?

File: src/NHSD.BuyingCatalogue.Solutions.Application/Commands/UpdateSolutionContactDetails/UpdateSolutionContactDetailsExecutor.cs [hunk 2]
```diff
@@ -4,6 +4,7 @@
 using System.Threading.Tasks;
 using NHSD.BuyingCatalogue.Solutions.Application.Commands.Execution;
 using NHSD.BuyingCatalogue.Solutions.Application.Persistence;
+using NHSD.BuyingCatalogue.Solutions.Application.Queries.GetContactDetailBySolutionId;
 using NHSD.BuyingCatalogue.Solutions.Application.Queries.GetSolutionById;
```
Comment (sam-roberts-bjss) [inline src/NHSD.BuyingCatalogue.Solutions.Application/Commands/UpdateSolutionContactDetails/UpdateSolutionContactDetailsExecutor.cs]: Is this using statement still referenced

File: src/NHSD.BuyingCatalogue.Solutions.API/Controllers/ContactDetailsController.cs (hunk 1 above)
Comment (assoomro) [reply src/NHSD.BuyingCatalogue.Solutions.API/Controllers/ContactDetailsController.cs]: 👍

File: src/NHSD.BuyingCatalogue.Solutions.Application/Commands/UpdateSolutionContactDetails/UpdateSolutionContactDetailsExecutor.cs (hunk 2 above)
Comment (assoomro) [reply src/NHSD.BuyingCatalogue.Solutions.Application/Commands/UpdateSolutionContactDetails/UpdateSolutionContactDetailsExecutor.cs]: 👍

**Pull request merged**